lib/network_monitor.c

```diff
@@ -682,7 +734,7 @@ static void notify_new_address(void *data, void *user_data)
 
         if (ops->new_address)
                 ops->new_address(ai->interface,
-                                 ai->address,
+                                 (const struct sockaddr *)&ai->address,
```

Comment:
mptcpd coding style places the CV qualifier to the right of the base type, i.e. `struct sockaddr const *` in this case.
```suggestion
                                 (struct sockaddr const *)&ai->address,
```